packages/lib/src/dataModel/BaseDataModel.ts

```diff
@@ -163,7 +163,7 @@ export abstract class BaseDataModel<TProps extends ModelProps> {
 setBaseDataModel(BaseDataModel)
 
 /**
- * @ignore
+ * @internal
```

Comment:
this cannot be internal since it is used as part of the definition of the "public" interface _DataModel